app/form/v1_0/fields/health-wellbeing.ts

```diff
@@ -20,6 +20,9 @@ const positiveFactorsHint = `
 <p class="govuk-hint">Consider what's helped them feel more hopeful.</p>
 <p class="govuk-hint">Select all that apply.</p>
 `
+const skipQuestionText = `
```

Comment:
If there is a better way to do this lmk! I tried a few ways to get the spacing between the two sentences and this worked best from what i found at the time.

Reply:
I think this works well enough 👍